src/test/java/plugins/GitLabPluginTest.java

```diff
@@ -0,0 +1,172 @@
+package plugins;
+
+import jakarta.inject.Inject;
+import org.gitlab4j.api.GitLabApiException;
+import org.jenkinsci.test.acceptance.Matchers;
+import org.jenkinsci.test.acceptance.docker.DockerContainerHolder;
+import org.jenkinsci.test.acceptance.docker.fixtures.GitLabContainer;
+import org.jenkinsci.test.acceptance.junit.*;
+import org.jenkinsci.test.acceptance.plugins.credentials.CredentialsPage;
+import org.jenkinsci.test.acceptance.plugins.credentials.ManagedCredentials;
+import org.jenkinsci.test.acceptance.plugins.gitlab_plugin.*;
+import org.jenkinsci.test.acceptance.po.Build;
+import org.jenkinsci.test.acceptance.po.WorkflowJob;
+import org.jenkinsci.test.acceptance.po.WorkflowMultiBranchJob;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.net.http.HttpResponse;
+
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+import static org.junit.Assert.*;
+
+@WithDocker
+@Category(DockerTest.class)
+@WithPlugins({"gitlab-branch-source", "workflow-multibranch"})
+public class GitLabPluginTest extends AbstractJUnitTest {
+
+    @Inject
+    DockerContainerHolder<GitLabContainer> gitLabServer;
+
+    private GitLabContainer container;
+    private String repoUrl;
+    private String host;
+    private int port;
+
+    private String privateTokenAdmin;
+
+    private String privateTokenUser;
+
+    private String repoName = "testrepo";
+
+    private String adminUserName = "testadmin";
+
+    private String userName = "testsimple";
+
+    public String getPrivateTokenAdmin() {
+        return privateTokenAdmin;
+    }
+
+    public String getPrivateTokenUser() {
+        return privateTokenUser;
+    }
+    @Before
+    public void init() throws InterruptedException, IOException {
+        container = gitLabServer.get();
+        repoUrl = container.getRepoUrl();
+        host = container.host();
+        port = container.port();
+        container.waitForReady(this);
+
+        // create an admin user
+        privateTokenAdmin = container.createUserToken(adminUserName, "arandompassword12#", "testadmin@example.com", "true");
+
+        // create another user
+        privateTokenUser = container.createUserToken(userName, "passwordforsimpleuser12#", "testsimple@example.com", "false");
+    }
+
+    @Test
+    public void dummy_test() {
```

Comment:
nit: Probably don't need this test anymore? Or I guess it can be kept as a smoke test

Reply:
I was also thinking to delete this test, just didn't get to it, I am deleting it now